packages/gatsby/src/commands/develop-process.ts

```diff
@@ -419,10 +419,10 @@ module.exports = async (program: IProgram): Promise<void> => {
   }
 
   // Start bootstrap process.
-  const { graphqlRunner } = await bootstrap(program)
+  const { bootstrapGraphQLFunction } = await bootstrap({ program })
```

Comment:
graphqlRunner has some state on itself - creating new one will cause additional (unneeded) work being done (related to materialization etc). Creating one and reusing it is good and we should do it, I think just the naming is suspect here - it's not really `bootstrapGraphQLFunction` - it is `gatsbyNodeGraphqlFunction` (as it's passed to `onPostBuild` etc which definitely is not in bootstrap). Maybe the way it's being passed around is also confusing - maybe there should be singleton and instead of passing it around we reach for it whenever we need it?

Reply:
> Maybe the way it's being passed around is also confusing - maybe there should be singleton and instead of passing it around we reach for it whenever we need it?

Tho I wouldn't do any of that in this PR (or maybe at all even)

Reply:
Currently it's for inc builds purposes

Reply:
I definitely prefer `gatsbyNodeGraphqlFunction`

Reply:
Or maybe `apiRunnerGraphqlFunction`. Any other bikesheds?